pyaedt/maxwell.py

```diff
@@ -1284,9 +1294,9 @@ def assign_force(self, input_object, reference_cs="Global", is_virtual=True, for
             self.logger.error("Solution Type has not Matrix Parameter")
```

Comment:
```suggestion
            self.logger.error("Solution type has no 'Matrix' parameter.")
```
The message should be a complete sentence. This suggestion assumes that "Matrix" is a parameter name.